AutomatticTracks/build.gradle

```diff
@@ -20,23 +20,22 @@ dependencies {
 
     testImplementation 'junit:junit:4.13.2'
     testImplementation 'org.assertj:assertj-core:3.19.0'
-    testImplementation 'org.mockito.kotlin:mockito-kotlin:2.2.11'
-    testImplementation 'org.mockito:mockito-inline:3.8.0'
+    testImplementation 'org.mockito.kotlin:mockito-kotlin:4.0.0'
+    testImplementation 'org.mockito:mockito-inline:4.6.1'
```

Comment:
Question (❓): Why did you decide to update to this version of `mockito-inline` and not its latest version? FYI: I am seeing that the latest version of this library is the [4.8.1](https://mvnrepository.com/artifact/org.mockito/mockito-inline).

PS: The same question (❓) applies to both, the `junit` and `runner` updates below, which are related to [androidx.test](https://developer.android.com/jetpack/androidx/releases/test), with their latest versions being the [1.1.4](https://developer.android.com/jetpack/androidx/releases/test#ext.junit-1.1.4) and [1.5.0](https://developer.android.com/jetpack/androidx/releases/test#runner-1.5.0) respectively.

Reply:
>  Why did you decide to update to this version of mockito-inline and not its latest version?

I created this PR some time ago (20th Jul) and then it was probably the newest version 😅 . Thanks for checking this!